### DISCUSSION TITLE
Spreading the word and user question

### DISCUSSION THREAD

**Discussion by sirinath** [jzimmerman/langcc#14] [General]
Following are some posts on this project.
- https://forum.devtalk.com/t/langcc-a-next-generation-compiler-compiler/75148
- https://www.reddit.com/r/ProgrammingLanguages/comments/xmmnu8/langcc_a_nextgeneration_compiler_compiler/
- https://www.reddit.com/r/computerscience/comments/xmmmcd/langcc_a_nextgeneration_compiler_compiler/
- https://www.reddit.com/r/opensource/comments/xmo7hu/langcc_a_nextgeneration_compiler_compiler/
- https://www.reddit.com/r/compsci/comments/xmmm7e/langcc_a_nextgeneration_compiler_compiler/
- https://lobste.rs/s/rkmmjc/langcc_next_generation_compiler
- https://news.ycombinator.com/item?id=32949019
- https://news.ycombinator.com/item?id=32910418

There are quite a few question which have been raised which you @jzimmerman might be only able to reply.

**Comment by jzimmerman** [comment]
Thanks @sirinath - I'll take a look!

**Reply by windenntw** [reply]
@jzimmerman Meta II & TreeMeta fan here... fantastic work! Specially because of not only doing the abstract theoretical part but also making sure that it works for real languages that people use daily :)

**Reply by jzimmerman** [reply]
Thank you! Glad you're finding langcc helpful.

**Comment by sirinath** [comment]
This might be of interest: https://nanopass.org
